src/components/ScholarshipsMadeSimple.jsx

```diff
@@ -0,0 +1,114 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import { Container } from '@material-ui/core';
+import { makeStyles } from '@material-ui/core/styles';
+import AppBar from '@material-ui/core/AppBar';
+import Tab from '@material-ui/core/Tab';
+import TabPanel from '@material-ui/lab/TabPanel';
+import TabContext from '@material-ui/lab/TabContext';
+import TabList from '@material-ui/lab/TabList';
+import Button from '@material-ui/core/Button';
+import Typography from '@material-ui/core/Typography';
+import PropTypes from 'prop-types';
+import LoginButton from './LoginButton';
+import HomeSection from './HomeSection';
+import pic from '../img/blank.png';
+
+const useStyles = makeStyles(() => ({
+  container: {
+    background: 'theme.palette.background.paper',
```

Comment:
It's not a string, it comes from the theme:
```suggestion
const useStyles = makeStyles((theme) => ({
  container: {
    background: theme.palette.background.paper,
```

Reply:
done